GDJS/tests/tests/asynctasks.js

```diff
@@ -0,0 +1,119 @@
+function createMockCallback() {
```

Comment:
If possible, put this into a describe as otherwise this is put in the global namespace